device/mellanox/x86_64-mlnx_msn4700-r0/ACS-MSN4700/port_config.ini

```diff
@@ -0,0 +1,33 @@
+# name          lanes                               alias
+Ethernet0       0,1,2,3,4,5,6,7                     etp1
```

Comment:
you should be able to split to 8x50 meaning that your interfaces should be Ethernet0, Ethernet8, etc.

Reply:
revised.